# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/audioconduit_unittests.cpp
# Commit: cca078b54e99
# Full Hash: cca078b54e99abd30cd3ff6fd5c1350e67ffc1b7
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 2
# Description:
#   Bug 1654112 - Avoid `using namespace` in headers. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112846
# ==============================================================================

diff -r 400d8a0f1a7d -r cca078b54e99 media/webrtc/signaling/gtest/audioconduit_unittests.cpp
--- a/media/webrtc/signaling/gtest/audioconduit_unittests.cpp	Tue Apr 06 17:45:07 2021 +0200
+++ b/media/webrtc/signaling/gtest/audioconduit_unittests.cpp	Tue Apr 06 18:30:27 2021 +0200
@@ -14,6 +14,7 @@
 
 using namespace mozilla;
 using namespace testing;
+using namespace webrtc;
 
 namespace test {
 